src/components/lyne-clock/lyne-clock.tsx

```diff
@@ -0,0 +1,209 @@
+import {
+  Component,
+  Element,
+  h,
+  Prop
+} from '@stencil/core';
+
+import clockFaceSVG from './assets/sbb_clock_face.svg';
+import clockHandleHoursSVG from './assets/sbb_clock_hours.svg';
+import clockHandleMinutesSVG from './assets/sbb_clock_minutes.svg';
+import clockHandleSecondsSVG from './assets/sbb_clock_seconds.svg';
+
+let moveMinuteHand;
+
+@Component({
+  shadow: true,
+  styleUrl: 'lyne-clock.scss',
+  tag: 'lyne-clock'
+})
+
+export class LyneClock {
+
+  @Prop() public paused = false;
+  @Prop() public initialtime = 'now';
```

Comment:
Especially here, a Prop description makes sense, to let users now what values they can pass in.
```
/** Pass a string with hours:minutes:seconds as a starting time for the clock, like 04:15:59 */
@Prop() public initialtime? = 'now';
```